CMakeLists.txt

```diff
@@ -178,15 +178,15 @@ target_link_libraries (mongocrypt PRIVATE kms_message_static)
 
 generate_export_header(mongocrypt EXPORT_FILE_NAME src/mongocrypt-export.h BASE_NAME mongocrypt )
 
-add_library (mongocrypt_static STATIC ${MONGOCRYPT_SOURCES} $<TARGET_OBJECTS:kms_message_obj>)
+add_library (mongocrypt_static STATIC ${MONGOCRYPT_SOURCES})
 target_include_directories (mongocrypt_static PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/kms-message/src")
 target_include_directories (mongocrypt_static PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/src>)
 target_include_directories (mongocrypt_static PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/src")
 target_include_directories (mongocrypt_static PRIVATE ${BSON_INCLUDES})
 target_compile_definitions (mongocrypt_static PRIVATE ${BSON_DEFINITIONS})
 target_link_libraries (mongocrypt_static PRIVATE ${BSON_TARGET})
 target_link_libraries (mongocrypt_static PRIVATE ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
Add a line here so that kms_message_static is included in mongocrypt_static:
`target_link_libraries (mongocrypt_static PRIVATE kms_message_static)`
Then you will only have to include mongocrypt_static in consumers.

Then you can remove the separate inclusion of `kms_message_static` below.

Reply:
done